scripts/create_playlist.py

```diff
@@ -0,0 +1,30 @@
+"""
+create_playlist.py
+creates a playlist in user librabry and returns the playlist id
+parameters: auth, list of tracks to add 
+"""
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+from .spotifyoauth import get_token
+from ..common import session
+
+def create_playlist(songs):
+    #get token 
+    session['token_info'], authorized = get_token()
+    if authorized:
```

Comment:
LGTM overall! Quick question - what happens if the user isn't authorized? (Not sure if this'll happen in practice; just curious about error handling, and if we can/should return an error if `authorized` is null or false)